src/links/dto/create-link.dto.ts

```diff
@@ -1,4 +1,5 @@
 import { IsUrl } from 'class-validator';
+import { IsSafeUrl } from 'src/decorators/safe-url.decorator';
```

Comment:
**praise**: Importing `IsSafeUrl` decorator enhances URL validation by adding an extra layer of security.